### PR TITLE
Fixes #2673 NTR CxG type IIx muscle cell

### LINKED ISSUE #2673: [NTR-cxg]type IIx muscle cell
I am currently working with the author of https://doi.org/10.1101/2024.05.15.594276. (@sebozzie)

**Preferred term label**
type IIx muscle cell

**Synonym(s)**
type 2x muscle cell, type 2x muscle fiber, type IIx muscle fiber

**Definition**
A fast type II muscle cell that utilize both the oxidative and glycolytic pathways for metabolism. These cells are distinguished by the expression of myosin heavy chain 1 (MYH1).

**Parent cell type term** 
CL:0002212: type II muscle cell

**Anatomical structure where the cell type is found**
UBERON:0001134: skeletal muscle tissue

**Additional notes or concerns**

- https://doi.org/10.1101/2024.05.15.594276:  "Within the myonuclei population type IIB, IIX and IIA myonuclei were distinguishable based on the expression of Myh4, Myh1 and Myh2, respectively"
- https://doi.org/10.1186/s13395-021-00279-0: "We provide and analyze a resource dataset based on relatively pure fibers, containing at least 80% of either MYH7 (marker of slow type 1 fibers), MYH2 (marker of fast 2A fibers), or MYH1 (marker of fast 2X fibers)."

### PR DESCRIPTION
**Pull request by aleixpuigb**
Fixes #2673 NTR CxG type IIx muscle cell
This PR solves problems of #2729

### CODE REVIEW

**Comment by Caroline-99** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -33131,6 +33132,19 @@ AnnotationAssertion(rdfs:label obo:CL_4052027 "atretic theca cell")
 SubClassOf(obo:CL_4052027 obo:CL_4033089)
 SubClassOf(obo:CL_4052027 ObjectSomeValuesFrom(obo:RO_0002202 obo:CL_0000503))
 
+# Class: obo:CL_4052026 (type IIx muscle cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727990") Annotation(oboInOwl:hasDbXref "PMID:35908794") Annotation(oboInOwl:hasDbXref "PMID:36361732") Annotation(oboInOwl:hasDbXref "doi:10.1101/2024.05.15.594276") obo:IAO_0000115 obo:CL_4052026 "A fast type II muscle cell that is part of the skeletal muscle tissue. This cell is characterized by its intermediate metabolic profile, utilizing both glycolytic and oxidative pathways for energy production. In humans, it is distinguished by the expression of myosin heavy chain 1 (MYH1).")
+AnnotationAssertion(terms:contributor obo:CL_4052026 <https://orcid.org/0009-0000-8480-9277>)
+AnnotationAssertion(terms:date obo:CL_4052026 "2024-11-04T10:26:29Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727990") oboInOwl:hasExactSynonym obo:CL_4052026 "type 2x muscle cell")
```
Comment (AvolaAmg) [inline src/ontology/cl-edit.owl:33140]: Consider adding the synonym: type 2x fiber 
from PMID: [34727990](https://pubmed.ncbi.nlm.nih.gov/34727990/)

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -33131,6 +33132,19 @@ AnnotationAssertion(rdfs:label obo:CL_4052027 "atretic theca cell")
...
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727990") oboInOwl:hasExactSynonym obo:CL_4052026 "type 2x muscle fiber")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1101/2024.05.15.594276") oboInOwl:hasExactSynonym obo:CL_4052026 "type IIx muscle fiber")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727990") rdfs:comment obo:CL_4052026 "In humans, type IIx fiber contains high levels of MYLK2, enhancing contraction speed and force via myosin phosphorylation, especially under low calcium. This mechanism supports post-activation potentiation, increasing force output in fast-twitch muscles. Additionally, this fiber has abundant ACTN3 (α-actinin-3), linked to muscle power, and exhibits the highest shortening velocity among human muscle fiber types.")
```
Comment (AvolaAmg) [inline src/ontology/cl-edit.owl:33143]: Consider adding in the extended description what are these fibers known for: sarcopenia, diabetes/obesity and Duchenne muscular dystrophy.
Apparently in loss of muscle fibres due to aging (sarcopenia) type 2x fibers are the ones that are missing the most
https://doi.org/10.1016/j.exger.2020.111171
PMID: 15462613
PMID: [27199166](https://pubmed.ncbi.nlm.nih.gov/27199166/)
also from this reference PMID: [27199166](https://pubmed.ncbi.nlm.nih.gov/27199166/) type 2x fibers increase in proportion in diabetes type 2 and obesity and they are the first one to degenerate during Duchenne Muscular Dystrophy.

**Review by AvolaAmg — CHANGES_REQUESTED** [review]

**Comment by AvolaAmg** [comment]
Please see in line comments!

**Comment by Caroline-99** [comment]
#gogoeditdiff

**Review by AvolaAmg — APPROVED** [review]
All good! thanks for addressing the comments

**Pull request merged**